src/main/scala/org/graphframes/lib/AggregateMessages.scala

```diff
@@ -31,19 +31,19 @@ import org.graphframes.GraphFrame
  *
  * For each triplet (source vertex, edge, destination vertex) in [[GraphFrame.triplets]],
  * this can send a message to the source and/or destination vertices.
- *  - [[AggregateMessages.Builder.sendToSrc()]] sends a message to the source vertex of each
+ *  - [[AggregateMessages.sendToSrc()]] sends a message to the source vertex of each
  *    triplet
- *  - [[AggregateMessages.Builder.sendToDst()]] sends a message to the destination vertex of each
+ *  - [[AggregateMessages.sendToDst()]] sends a message to the destination vertex of each
  *    triplet
- *  - [[AggregateMessages.Builder.agg()]] specifies an aggregation function for aggregating the
+ *  - [[AggregateMessages.agg()]] specifies an aggregation function for aggregating the
  *    messages sent to each vertex.  It also runs the aggregation, computing a DataFrame
  *    with one row for each vertex which receives > 0 messages.  The DataFrame has 2 columns:
  *     - vertex column ID (named [[GraphFrame.ID]])
  *     - aggregate from messages sent to vertex (with the name given to the [[Column]] specified
- *       in [[AggregateMessages.Builder.agg()]])
+ *       in [[AggregateMessages.agg()]])
  *
  * When specifying the messages and aggregation function, the user may reference columns using:
- *  - [[AggregateMessages.src]]: column for source vertex of edge
+ *  - [[AggregateMessages]]: column for source vertex of edge
```

Comment:
IntelliJ does not like links to a class' members within docs for the overall class.  I've never been sure how to get around that.